decision/15.md

```diff
@@ -3,40 +3,48 @@ published: true
 layout: default-theme-wet-boew-en
 filename_fr: 15-fr
 title: Design decision 15 - Removal of trailing slash for void elements (self-closing tags)
-description: Removal of trailing slash for void elements (self-closing tags) located in the head of Canada.ca templates.
-modified: 2022-11-16
+description: Removal of trailing slash for void elements (self-closing tags) located in any Canada.ca templates.
+modified: 2022-11-17
 ---
 
-Removal of trailing slash for [void elements](https://html.spec.whatwg.org/multipage/syntax.html#void-elements) (self-closing tags) located in the head of Canada.ca templates.
+Removal of trailing slash for [void elements](https://html.spec.whatwg.org/multipage/syntax.html#void-elements) (self-closing tags) located in Canada.ca templates.
 
 ## Summary
 
 * **Design decision number:** 15
 * **Expert:** @Ricokola
-* **Submited:** 2022-11-16
+* **Submited:** 2022-11-17
 * **Validated:** by @[username] on YYYY-MM-DD
 * **Status:** Approved on YYYY-MM-DD
 * **Type:** HTML5 Markup validation
+* **Github discussion:** [Open discussion #9483](https://github.com/wet-boew/wet-boew/discussions/9483](https://github.com/wet-boew/wet-boew/discussions/9483)
+* **History:**
+	* (2022-11-21) Addition of new findings and evidence to help support our recommendation.
+	* (2022-11-21) Revised
```

Comment:
Those 2 item can be merged, because "revised" is kind the default meaning of an history item and both is referring to the same date.